crates/vm/levm/src/precompiles.rs

```diff
@@ -94,7 +96,10 @@ pub fn is_precompile(callee_address: &Address) -> bool {
 pub fn execute_precompile(current_call_frame: &mut CallFrame) -> Result<Bytes, VMError> {
     let callee_address = current_call_frame.code_address;
     let calldata = current_call_frame.calldata.clone();
-    let gas_for_call = current_call_frame.gas_limit;
+    let gas_for_call = current_call_frame
+        .gas_limit
+        .checked_sub(current_call_frame.gas_used)
```

Comment:
can `gas_used` at this point be other than zero?

Reply:
In create transactions that called a precompiled contract, the gas_limit was not being affected by intrinsic gas, and this fix is ​​a patch for that. Maybe we could handle that case in a better way.